schedule/yast/sle/flows/default_slem.yaml

```diff
@@ -4,7 +4,8 @@ bootloader:
   - installation/bootloader_start
 setup_libyui:
   - installation/setup_libyui
-access_beta_distribution: []
+# access_beta_distribution:
```

Comment:
Why not removed and just comments here?

Reply:
so we just need to un-comment when when we have beta again